packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -1,45 +1,92 @@
+import styled from '@emotion/styled'
 import graphql from 'babel-plugin-relay/macro'
 import React, {Suspense} from 'react'
-import {Link} from 'react-router-dom'
-
-import { TeamPromptMeeting_meeting$key } from '~/__generated__/TeamPromptMeeting_meeting.graphql'
+import {commitLocalUpdate, useFragment} from 'react-relay'
+import useAtmosphere from '~/hooks/useAtmosphere'
+import useMeeting from '~/hooks/useMeeting'
+import {DiscussionThreadEnum} from '~/types/constEnums'
+import {TeamPromptMeeting_meeting$key} from '~/__generated__/TeamPromptMeeting_meeting.graphql'
 import logoMarkPurple from '../styles/theme/images/brand/mark-color.svg'
+import LinkButton from './LinkButton'
 import MeetingArea from './MeetingArea'
 import MeetingContent from './MeetingContent'
 import MeetingHeaderAndPhase from './MeetingHeaderAndPhase'
 import MeetingStyles from './MeetingStyles'
-import TeamPromptTopBar from './TeamPrompt/TeamPromptTopBar'
 import PhaseWrapper from './PhaseWrapper'
-import {useFragment} from 'react-relay'
+import TeamPromptDiscussionDrawer from './TeamPrompt/TeamPromptDiscussionDrawer'
+import TeamPromptTopBar from './TeamPrompt/TeamPromptTopBar'
 
 interface Props {
   meeting: TeamPromptMeeting_meeting$key
 }
 
+const StyledMeetingHeaderAndPhase = styled(MeetingHeaderAndPhase)<{isOpen: boolean}>(
+  ({isOpen}) => ({
+    width: isOpen ? `calc(100% - ${DiscussionThreadEnum.WIDTH}px)` : '100%'
+  })
+)
+
 const TeamPromptMeeting = (props: Props) => {
   const {meeting: meetingRef} = props
+  const atmosphere = useAtmosphere()
   const meeting = useFragment(
     graphql`
       fragment TeamPromptMeeting_meeting on TeamPromptMeeting {
+        ...useMeeting_meeting
         ...TeamPromptTopBar_meeting
+        ...TeamPromptDiscussionDrawer_meeting
+        id
+        isRightDrawerOpen
+        phases {
+          stages {
+            ... on TeamPromptResponseStage {
+              discussionId
+            }
+          }
+        }
       }
     `,
     meetingRef
   )
+  const {safeRoute, isDesktop} = useMeeting(meeting)
+
+  const {isRightDrawerOpen} = meeting
+
+  const selectDiscussion = () => {
+    // :TODO: (jmtaber129): Get the discussionId from the response card that was clicked.
+    const {phases, id: meetingId} = meeting
+    const phase = phases[0]
+    const {stages} = phase!
+    const stage = stages[0]
+    const {discussionId} = stage!
+
+    commitLocalUpdate(atmosphere, (store) => {
+      const meetingProxy = store.get(meetingId)
+      if (!meetingProxy) return
+      meetingProxy.setValue(discussionId, 'localDiscussionId')
```

Comment:
-1 I might be misunderstanding this, but could we just use `stage.discussionId`? 
That way the url is the single source of truth (stageId) and when someone clicks a different response, we just navigate to a different stageId, which gives us a different `stage.discussionId`.

Reply:
The only issue I see with this approach is when the drawer is closed, but can be reopened without clicking on a discussion - it seems wrong to include the stage ID in the URL when there's no discussion currently open.

Reply:
agreed, that does feel weird.
could we combine the open drawer boolean with the stage number?
the use case I'm trying to solve for is the, "hey, let's look at my discussion thread. let me send you a URL so you can see exactly what i see."

e.g.
```
/responses -> right drawer closed
/responses/1-> right drawer open to the first user
/responses/2 -> right darwer open to the second user
click the drawer close button -> goto /responses
click the drawer open button -> goto /responses/1
```

this uses numbers instead of a stageId, i don't have a strong opinion either way.  the other meetings use a number, but if we do use numbers then we'll need 2 sort orders: the order that comes from the server & the order in which they are displayed (ie the viewer should always be the first item so they can edit their card without it getting moved when someone new joins the team)

Reply:
The behavior you mentioned seems much better, though I do think it would be better to use stageId for simplicity.

I think for this, I'll make the following changes:
* Use `localStageId` instead of `localDiscussionId`
* Keep internal relay state as the source of truth, and update the URL based on local stage ID + whether the drawer is open
* Initialize internal relay state based on URL (i.e. if there's a stage ID in the URL, set it as the internal stage ID and set the drawer to open).

This particular change is starting to feel like a bit more scope than originally intended, and possibly more coupled with the changes needed to integrate discussion threads with response cards.  Thoughts on creating a new task for this or adding this to the AC of https://github.com/ParabolInc/parabol/issues/6372 instead of tackling it as part of this PR?

Reply:
sure, it's all locked behind a feature flag so any way you wanna break it up is OK with me. we can merge this chunk in as soon as that merge conflict is resolved